README.md

```diff
@@ -10,44 +10,137 @@
   Any changes to README.md directly will result in a failure on CI.
 -->
 
-# Angular CLI
-### Development tools and libraries specialized for Angular
+<h1 align="center">Angular CLI - The CLI tool for Angular.</h1>
 
-This is the home of the DevKit and the Angular CLI code. You can find the Angular CLI specific README
-[here](/packages/angular/cli/README.md).
+<p align="center">
+  <br>
+  <img src="/docs/images/angular-cli-logo.png" alt="Angular CLI logo" width="100px" height="108px"/>
+  <br><br>
+  <i>The Angular CLI is a command-line interface tool that you use to initialize, develop, scaffold, 
+    <br>and maintain Angular applications directly from a command shell.</i>
+  <br>
+</p>
 
+<p align="center">
+  <a href="https://cli.angular.io"><strong>cli.angular.io</strong></a>
```

Comment:
I wonder if we should point people at angular.io/cli instead? This subdomain redirects there already and I don't think we have any plans to do anything with it.

Reply:
I don’t have any strong objects. Although in the angular/angular readme cli.angular.io is used.